tests/celeborn-spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala

```diff
@@ -58,22 +57,23 @@ trait SparkTestBase extends Logging with MiniClusterFeature {
           Worker,
           RpcEnv,
           Worker,
+          RpcEnv,
           Thread,
           Thread,
           Thread,
-          Thread,
-          ChannelFuture)): Unit = {
+          Thread)): Unit = {
     tuple._3.close()
     tuple._4.shutdown()
     tuple._5.close()
     tuple._6.shutdown()
     tuple._7.close()
-    tuple._1.stop()
+    tuple._8.shutdown()
+    tuple._1.close()
```

Comment:
it's fragile to access variable by index, we should refractor it later

Reply:
+1